lib/cmock_generator_plugin_cexception.rb

```diff
@@ -12,6 +12,7 @@ def initialize(config, utils)
     @config = config
     @utils = utils
     @priority = 7
+    raise 'Error: cexception is not supported without setjmp support' unless @config.has_setjmp_h
```

Comment:
I think this needs to switch to exclude?

Reply:
You are indeed correct. Will fix these now.